autoload/terminal_images.vim

```diff
@@ -654,3 +654,26 @@ function! terminal_images#ToggleGlobal() abort
         call terminal_images#EnableGlobal()
     endif
 endfun
+
+function! terminal_images#EnableBuffer() abort
```

Comment:
One of my concerns is that this function may be a bit confusing since it enables image display in a buffer even if it's disabled globally. Probably worth adding a comment mentioning that.

Reply:
Added a coment